PhpAmqpLib/Wire/IO/StreamIO.php

```diff
@@ -210,7 +210,7 @@ public function read($len)
                 if ($this->canDispatchPcntlSignal) {
                     // prevent cpu from being consumed while waiting
                     if ($this->canSelectNull) {
-                        $this->select(null, null);
+                        $this->select($this->heartbeat ?  $this->heartbeat + 1 : null, null);
```

Comment:
Can you pull `$this->heartbeat ?  $this->heartbeat + 1 : null` out into a variable.  I think it would increase readability.

Something like:
```php
$timeoutSeconds = $this->heartbeat ?  $this->heartbeat + 1 : null;
$this->select($timeoutSeconds, null);
```